.github/workflows/lint-rust.yml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     paths:
       - crates/**
+      - docs/src/rust/**
```

Comment:
Maybe it is good to keep this one in a separate workflow that only triggers if we make changes there?

Reply:
Not exactly sure what you mean?

With this change, it will run `clippy` etc. whenever we make changes there.

If we do not make changes there, nothing will be different!

Reply:
Ok!